src/components/Controls/Controls.tsx

```diff
@@ -22,18 +22,18 @@ const b = block('dc-controls');
 
 export interface ControlsProps {
```

Comment:
Won't it break if only required props will be passed? (without `hideControls`)

Reply:
This won't break
Only an empty div container will be rendered if only `lang` is passed